### DISCUSSION TITLE
[SOLVED] v2.0.0-beta.1

### DISCUSSION THREAD

**Discussion by pioardi** [poolifier/poolifier#190] [Show and tell]
I created  a new release to publish, but our GitHub action failed on npm run lint command.

    - https://github.com/pioardi/poolifier/runs/1930013903?check_suite_focus=true

@Shinigami92  could you please look into that?
FYI @jerome-benoit

**Comment by pioardi** [comment]
Should we use src instead of lib into our tests and benchmarks ?

**Reply by jerome-benoit** [reply]
https://github.com/pioardi/poolifier/pull/192
